contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -1145,11 +1145,11 @@ reduce data volume, load on the system, and noise exposed to users.
 
 ## Naming conventions
 
-* Go field names must be CamelCase. JSON field names must be camelCase. Other
+* Go field names must be Pascal case. JSON field names must be camelCase. Other
 than capitalization of the initial letter, the two should almost always match.
```

Comment:
While a slight improvement, this remains confusing. The copy violates conventional orthography (permissibly, I say) to demonstrate the setting, "camelCase." in other words, "camelCase" is set in camel case. Pascal case, on the other hand, is set conventionally, not in Pascal case (i.e., "PascalCase"). An untutored reader will naturally assume that camel case is set with one capital in the middle, whereas Pascal case has an initial cap and a space in the middle, which is not so. Either set Pascal case as "PascalCase" or set camel case normally and explain both or link to a good explanation.

Reply:
I agree and I'd prefer to see: **PascalCase** to limit confusion